{
  "Header": {
    "Original": "CVE-2020-27187 (An issue was discovered in KDE Partition Manager 4.1.0 before 4.2.0. T ...)",
    "Line": 248857,
    "ID": "CVE-2020-27187",
    "Description": "(An issue was discovered in KDE Partition Manager 4.1.0 before 4.2.0. T ...)"
  },
  "Annotations": [
    {
      "Original": "- kpmcore 4.2.0-1",
      "Line": 248858,
      "Type": "package",
      "Package": "kpmcore",
      "Kind": "fixed",
      "Version": "4.2.0-1"
    },
    {
      "Original": "[buster] - kpmcore \u003cnot-affected\u003e (kpmcore_externalcommand not yet present)",
      "Line": 248859,
      "Type": "package",
      "Release": "buster",
      "Package": "kpmcore",
      "Kind": "not-affected",
      "Description": "kpmcore_externalcommand not yet present"
    },
    {
      "Original": "[stretch] - kpmcore \u003cnot-affected\u003e (kpmcore_externalcommand not yet present)",
      "Line": 248860,
      "Type": "package",
      "Release": "stretch",
      "Package": "kpmcore",
      "Kind": "not-affected",
      "Description": "kpmcore_externalcommand not yet present"
    },
    {
      "Original": "NOTE: https://kde.org/info/security/advisory-20201017-1.txt",
      "Line": 248861,
      "Type": "NOTE",
      "Description": "https://kde.org/info/security/advisory-20201017-1.txt"
    },
    {
      "Original": "NOTE: https://invent.kde.org/system/kpmcore/-/commit/c466c5db11b5cee546d1ec0594c2f1105a354fed (fix)",
      "Line": 248862,
      "Type": "NOTE",
      "Description": "https://invent.kde.org/system/kpmcore/-/commit/c466c5db11b5cee546d1ec0594c2f1105a354fed (fix)"
    },
    {
      "Original": "NOTE: https://invent.kde.org/system/kpmcore/-/commit/7ec4b611dcf822439b081613cca4184689266454 (removes KF5 5.73 dependency)",
      "Line": 248863,
      "Type": "NOTE",
      "Description": "https://invent.kde.org/system/kpmcore/-/commit/7ec4b611dcf822439b081613cca4184689266454 (removes KF5 5.73 dependency)"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1890199",
      "Line": 248864,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1890199"
    }
  ]
}